htdocs/install/mysql/tables/llx_c_productbatch_status.sql

```diff
@@ -0,0 +1,25 @@
+-- ========================================================================
+-- Copyright (C) 2012-2017      Noé Cendrier  <noe.cendrier@altairis.fr>
+--
+-- This program is free software; you can redistribute it and/or modify
+-- it under the terms of the GNU General Public License as published by
+-- the Free Software Foundation; either version 2 of the License, or
+-- (at your option) any later version.
+--
+-- This program is distributed in the hope that it will be useful,
+-- but WITHOUT ANY WARRANTY; without even the implied warranty of
+-- MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+-- GNU General Public License for more details.
+--
+-- You should have received a copy of the GNU General Public License
+-- along with this program. If not, see <http://www.gnu.org/licenses/>.
+--
+-- ========================================================================
+
+create table llx_c_productbatch_status
```

Comment:
Can you rename table into
llx_c_productbatch_qcstatus
qcstatus = Quality control status 
The field status must be reserved for fields that store a hard coded status.